### PR TITLE
Dayseven

### PR DESCRIPTION
**Pull request by Talaspaev**

### CODE REVIEW

File: src/Components/Profile/MyPosts/Post/post.jsx [hunk 1]
```diff
@@ -5,16 +5,23 @@ import style from './post.module.css';
 import logo from './post.jpg';
 
 const Post = (props) => {
-  const { message } = props;
-  return (
-    <div className={style.item}>
+  const { profileData } = props.messages;
+  const PostData = Object.values(profileData);
```
Comment (Dmitry-White) [inline src/Components/Profile/MyPosts/Post/post.jsx]: The naming of this variable is confusing. The fact that it starts with capital letter implies that it's either a Class, a Constructor function or a Component.
But in reality it's just some data - so consider following the naming conventions and start the names for such variables with lowercase letter.
https://github.com/airbnb/javascript#naming-conventions

File: src/Components/Profile/MyPosts/myPosts.jsx [hunk 2]
```diff
@@ -2,21 +2,31 @@ import React from 'react';
...
-const MyPosts = () => (
-  <div>
+const MyPosts = (props) => {
+  const { updateTextaAreaFunc, textAreaFunc } = props.props;
+  const createRefFortext = React.createRef();
+  const addPost = () => {
+    const text = createRefFortext.current.value;
+    textAreaFunc(text);
+    updateTextaAreaFunc('');
+  };
```
Comment (Dmitry-White) [inline src/Components/Profile/MyPosts/myPosts.jsx]: Class methods should be separated with empty lines for readability.

File: src/redux/state.js [hunk 3]
```diff
@@ -1,19 +1,58 @@
...
+
   friends: [
     { id: 1, name: 'Dymuch', src: 'https://upload.wikimedia.org/wikipedia/commons/9/9f/Dmitry_Medvedev_2016.jpg' },
     { id: 2, name: 'Natalya', src: 'https://images11.cosmopolitan.ru/upload/img_cache/7b3/7b3b3da698f08bee6e9e7402053f6df5_ce_750x497x0x129_fitted_740x0.jpg' },
     { id: 3, name: 'Adil', src: 'https://img.gazeta.ru/files3/784/11560784/skrskr2-pic905-895x505-4108.jpg' }],
 };
-export default state;
+
+const textAreaFunc = (NewPost, id) => {
+  const PostData = {
```
Comment (Dmitry-White) [inline src/redux/state.js]: https://github.com/airbnb/javascript#naming-conventions

File: src/redux/state.js [hunk 4]
```diff
@@ -1,19 +1,58 @@
...
+  },
+
   friends: [
     { id: 1, name: 'Dymuch', src: 'https://upload.wikimedia.org/wikipedia/commons/9/9f/Dmitry_Medvedev_2016.jpg' },
     { id: 2, name: 'Natalya', src: 'https://images11.cosmopolitan.ru/upload/img_cache/7b3/7b3b3da698f08bee6e9e7402053f6df5_ce_750x497x0x129_fitted_740x0.jpg' },
     { id: 3, name: 'Adil', src: 'https://img.gazeta.ru/files3/784/11560784/skrskr2-pic905-895x505-4108.jpg' }],
 };
-export default state;
+
+const textAreaFunc = (NewPost, id) => {
```
Comment (Dmitry-White) [inline src/redux/state.js]: https://github.com/airbnb/javascript#naming-conventions

File: src/render.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-cycle */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from './App/App';
+
+import { textAreaFunc, updateTextaAreaFunc } from './redux/state';
+
+const renderFunc = (state) => {
+  ReactDOM.render(<App state={state} textAreaFunc={textAreaFunc} updateTextaAreaFunc={updateTextaAreaFunc} />, document.getElementById('root'));
```
Comment (Dmitry-White) [inline src/render.js]: Consider not making such long lines of code :)
This way it looks better and more readable.
image

File: src/Components/Profile/MyPosts/Post/post.jsx (hunk 1 above)
Comment (Talaspaev) [reply src/Components/Profile/MyPosts/Post/post.jsx]: I've read this text.It was very useful for me

File: src/redux/state.js (hunk 3 above)
Comment (Talaspaev) [reply src/redux/state.js]: same above

File: src/redux/state.js (hunk 4 above)
Comment (Talaspaev) [reply src/redux/state.js]: same above

File: src/render.js (hunk 5 above)
Comment (Talaspaev) [reply src/render.js]: Sure it's realy more readable

**Pull request merged**